src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, isMarried, partnerName, sex },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {name}</h2>
+
+    {age && <p className="Person__age">I am {age}</p>}
+    {isMarried && sex === 'm' && (
+      <p className="Person__partner">{partnerName} is my wife</p>
+    )}
+    {isMarried && sex === 'f' && (
+      <p className="Person__partner">{partnerName} is my husband</p>
+    )}
+    {!isMarried && <p className="Person__partner">I am not married</p>}
```

Comment:
Try using one condition 
IsMarried ? condition which render wife or husband : default text ( I am not.. )

Reply:
Fixed, thanks a lot)